packages/nextjs/app/builders/0x28482B1279E442f49eE76351801232D58f341CB9/components/SocialLinks.tsx

```diff
@@ -0,0 +1,40 @@
+"use client";
```

Comment:
You don't need to use "use client" here, as this can be server-rendered. There are no client-side actions like useEffect or useState involved.

Reply:
I've remove "use client"